dll/win32/imm32/ime.c

```diff
@@ -1229,9 +1229,9 @@ ImmGetConversionListA(HKL hKL, HIMC hIMC, LPCSTR pSrc, LPCANDIDATELIST lpDst,
 
 Quit:
     if (pszSrcW)
-        HeapFree(g_hImm32Heap, 0, pszSrcW);
+        Imm32HeapFree(pszSrcW);
     if (pCL)
-        HeapFree(g_hImm32Heap, 0, pCL);
+        Imm32HeapFree(pCL);
```

Comment:
NULL check is not required, since passing in `NULL` to `HeapFree` is allowed.
With this you can simplify quite some of these cases :)

Reply:
I omitted some NULL checks. Ok.